agent/bench-scripts/gold/pbench-user-benchmark/test-12.txt

```diff
@@ -4,15 +4,15 @@ ERROR:root:current position: 51
 Traceback (most recent call last):
   File "/var/tmp/pbench-test-bench/opt/pbench-agent/bench-scripts/postprocess/user-benchmark-wrapper", line XXX, in <module>
     data = json.load(data_file)
-  File "/usr/lib/python2.7/json/__init__.py", line XXX, in load
-    **kw)
-  File "/usr/lib/python2.7/json/__init__.py", line XXX, in loads
+  File "/usr/lib/python3.6/json/__init__.py", line XXX, in load
```

Comment:
Missed this before: this fails for me because I'm running 3.7.

Reply:
Fixed this by removing the exception entirely.  And did a better conversion to Python 3 while cleaning up the code in `user-benchmark-wrapper`.